persistence/src/androidTest/kotlin/de/cyface/persistence/content/LocationSerializerTest.kt

```diff
@@ -97,7 +97,7 @@ class LocationSerializerTest {
         testReadFrom(3600 * 10)
     }
 
-    private fun testReadFrom(numberOfTestEntries: Int) {
+    private fun testReadFrom(numberOfTestEntries: Int) = runBlocking {
```

Comment:
Required because we now correctly mark the database operations as suspend functions.
runBlocking is fine in tests.